ctapipe/resources/base_config.yaml

```diff
@@ -38,7 +38,12 @@ CameraCalibrator:
   #
   # Note this is a telescope-wise parameter, so can be specified per telescope
   # if necessary (see below for an example)
-  image_extractor_type: NeighborPeakWindowSum
+  image_extractor_type:
+    - ['type', '*', 'NeighborPeakWindowSum']
+    - ['type', '*FlashCam', 'FlashCamExtractor']
+
+  FlashCamExtractor:
+    exponential_scale: 7.5
```

Comment:
This option doesn't seem to exist in the Extractor - was it forgotten?

Reply:
I guess Max added this as a placeholder in our first discussion - not needed anymore.